task/anchore-cli/0.1/anchore-cli.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: anchore-cli
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, anchore
+    tekton.dev/displayName: "anchore cli"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+    Using the Anchore CLI users can manage and inspect images, policies, subscriptions and registries
+  workspaces:
+    - name: manifest-dir
+  params:
+    - name: ARGS
+      description: The Arguments to be passed to anchore command.
+      type: array
+    - name: ANCHORE_CLI_IMAGE
+      default: anchore/engine-cli
```

Comment:
can you please tag the image as well with it's appropriate tag and digest? (Reason: The integrity of Task should remaining the same everytime somebody uses the Task)

Reply:
sure will do it